modules/islandora_entities_csv_import/includes/islandora_entities_csv_import.form.inc

```diff
@@ -57,3 +58,42 @@ function islandora_entities_import_form_submit(array $form, array &$form_state)
   $collection_pid = $form_state['values']['collection'];
   islandora_entities_import_csv_batch(drupal_realpath($file->uri), $collection_pid);
 }
+
+/**
+ * Validates the collection Drupal Form Element.
+ *
+ * Ensures that the object exists, is a collection and has the correct policy.
+ *
+ * @param array $element
+ *   The collection Drupal Form Element.
+ * @param array $form_state
+ *   The Drupal Form State.
+ */
+function islandora_entities_csv_import_validate_collection(array $element, array &$form_state) {
+  $collection_pid = &$form_state['values']['collection'];
```

Comment:
@d-r-p just curiosity. Why are you loading the collection_pid by reference here?

Reply:
Bad copy&paste, I guess... I will remove it when cleaning up.